sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -67,9 +67,11 @@ synchronized void setMetrics(final @NotNull Activity activity, final @NotNull Se
     SparseIntArray[] framesRates = null;
     try {
       framesRates = frameMetricsAggregator.remove(activity);
-    } catch (IllegalArgumentException ignored) {
-      // it throws when attempt to remove OnFrameMetricsAvailableListener that was never added.
-      // there's no contains method
+    } catch (Exception ignored) {
```

Comment:
hm, should we rather do a multi-catch? cause now it will swallow all other exceptions as well?

Reply:
that's the goal, to avoid problems with the further version compatibility, since it's the 2nd, I'd rather prefer to debug once slow/frozen frames are missing rather than crashing the final app.